x-pack/plugins/ml/server/client/elasticsearch_ml.ts

```diff
@@ -671,7 +671,7 @@ export const elasticsearchJsPlugin = (Client: any, config: any, components: any)
     },
     method: 'DELETE',
   });
-
+  // Filters default size limit is 10 so explicitly set to 1000 for now.
```

Comment:
is `10` correct in this comment?

Reply:
No, it has now been fixed on the back-end to be `100`.

Reply:
'tis